### ISSUE TITLE
Display the abstention ratio (or count)

### ISSUE THREAD

**Issue by zigarn** [Zenika/humeur-du-mois-2018#169]
Having the number of potential voters each month would be great (if not necessary from my point of view) to have a real picture of the situation.

**Comment by hgwood** [comment]
Yes that would be useful. It would require some rework though. Currently the directory of voters is imported from Alibeez every day so that the app is aware of new people but there is no clean up so there's more voters in the app than actual potential voters.
